test/rekt/resources/trigger/trigger.go

```diff
@@ -19,20 +19,23 @@ package trigger
 import (
 	"context"
 	"embed"
+	"encoding/json"
 	"strings"
 	"time"
 
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	eventingv1 "knative.dev/eventing/pkg/apis/eventing/v1"
 	duckv1 "knative.dev/pkg/apis/duck/v1"
 	"knative.dev/reconciler-test/pkg/feature"
 	"knative.dev/reconciler-test/pkg/k8s"
 	"knative.dev/reconciler-test/pkg/manifest"
+	"sigs.k8s.io/yaml"
 
 	"knative.dev/eventing/test/rekt/resources/delivery"
 )
 
 //go:embed *.yaml
-var yaml embed.FS
+var yamlEmbed embed.FS
```

Comment:
so `yamlEmbed` is the yaml template right? What is the reason behind this change?

Reply:
Because we are importing the package "yaml" now, so there had been a name conflict